File: Iniciante/URI 1008.cs
using System; 
 
class URI {
 
    static void Main(string[] args) { 
        int funcionarios = int.Parse(Console.ReadLine());
        int horas = int.Parse(Console.ReadLine());
		double valorPorHora = double.Parse(Console.ReadLine());
		double salario = horas * valorPorHora;
        Console.WriteLine("NUMBER = {0}", funcionarios);
		Console.WriteLine("SALARY = U$ {0:F2}", salario);
    }
    
}

File: Iniciante/URI 1008.dart
import 'dart:io';

void main() {
  int funcionarios = int.parse(stdin.readLineSync());
  int horas = int.parse(stdin.readLineSync());
  double valorPorHora = double.parse(stdin.readLineSync());
  double salario = horas * valorPorHora;
  print("NUMBER = $funcionarios");
  print("SALARY = U\$ ${salario.toStringAsFixed(2)}");
}
